tool/tctl/common/plugin/awsic.go

```diff
@@ -202,6 +202,32 @@ func (p *PluginsCommand) InstallAWSIC(ctx context.Context, args installPluginArg
 		return trace.Wrap(err)
 	}
 
+	settings := &types.PluginAWSICSettings{
+		Region: awsICArgs.region,
+		Arn:    awsICArgs.arn,
+		ProvisioningSpec: &types.AWSICProvisioningSpec{
+			BaseUrl: awsICArgs.scimURL.String(),
+		},
+		AccessListDefaultOwners: awsICArgs.defaultOwners,
+		UserSyncFilters:         userFilters,
+		GroupSyncFilters:        groupFilters,
+		AwsAccountsFilters:      accountFilters,
+	}
+
+	if awsICArgs.useSystemCredentials {
+		settings.Credentials = &types.AWSICCredentials{
+			Source: &types.AWSICCredentials_System{
+				System: &types.AWSICCredentialSourceSystem{
+					AwsRoleArn: "",
```

Comment:
Don't we need to extend the CLI to support optional new input ?

Reply:
Yes. Coming up in next PR.